test/structure/test_structure.py

```diff
@@ -41,34 +38,41 @@ def test_structure_ctor_bad_input():
     assert exe.type == SystemExit
     assert exe.value.code == 1
 
+
 def test_residues():
     pdb_file_path = f"{DATA_DIR}1Q4T_ligand_test.pdb"
     stru: Structure = sp.get_structure(pdb_file_path)
 
     assert len(stru.residues) == 580
 
+
 def test_ligands():
     pdb_file_path = f"{DATA_DIR}1Q4T_ligand_test.pdb"
     stru: Structure = sp.get_structure(pdb_file_path)
     assert len(stru.ligands) == 2
     assert stru.ligands[0].name == "4CO"
 
+
 def test_solvents():
     pdb_file_path = f"{DATA_DIR}1Q4T_ligand_test.pdb"
     stru: Structure = sp.get_structure(pdb_file_path)
     assert len(stru.solvents) == 296
     assert stru.solvents[0].name == "HOH"
 
+
 def test_peptides():
     pdb_file_path = f"{DATA_DIR}1Q4T_ligand_test.pdb"
     stru: Structure = sp.get_structure(pdb_file_path)
     assert len(stru.peptides) == 2
-    assert tuple(map(lambda x: x.name, stru.peptides)) == ("A","B")
+    assert tuple(map(lambda x: x.name, stru.peptides)) == ("A", "B")
+
 
 def test_sequence():
     pdb_file_path = f"{DATA_DIR}1Q4T_ligand_test.pdb"
     stru: Structure = sp.get_structure(pdb_file_path)
-    assert stru.sequence["A"] == "ATGGNLPDVASHYPVAYEQTLDGTVGFVIDEMTPERATASVEVTDTLRQRWGLVHGGAYCALAEMLATEATVAVVHEKGMMAVGQSNHTSFFRPVKEGHVRAEAVRIHAGSTTWFWDVSLRDDAGRLCAVSSMSIAVRPRRD"
+    assert stru.sequence[
+        "A"] == "ATGGNLPDVASHYPVAYEQTLDGTVGFVIDEMTPERATASVEVTDTLRQRWGLVHGGAYCALAEMLATEATVAVVHEKGMMAVGQSNHTSFFRPVKEGHVRAEAVRIHAGSTTWFWDVSLRDDAGRLCAVSSMSIAVRPRRD"
```

Comment:
same as above. Any way to avoid this kind of change?

Reply:
Not possible to fix directly. You would have to have a crazy long line. Instead, I have made the variable `target_seq` on the preceding line.